db-tests/src/test/java/gov/nasa/jpl/aerie/database/PlanCollaborationTests.java

```diff
@@ -80,6 +80,10 @@ void beforeAll() throws SQLException, IOException, InterruptedException {
     helper.startDatabase();
     connection = helper.connection();
     merlinHelper = new MerlinDatabaseTestHelper(connection);
+    merlinHelper.insertUser("PlanCollaborationTests");
+    merlinHelper.insertUser("PlanCollaborationTests Reviewer");
```

Comment:
I'm curious about the intent behind inserting these users - e.g. "PlanCollaborationTests Reviewer" that seem unused at first glance

Reply:
All of these users are used during tests in this class:
- [PlanCollaborationTests](https://github.com/NASA-AMMOS/aerie/blob/5a9ffa15010924cad1e62ef206b08dfe3852aac3/db-tests/src/test/java/gov/nasa/jpl/aerie/database/PlanCollaborationTests.java#L111)
- [PlanCollaborationTests Requester](https://github.com/NASA-AMMOS/aerie/blob/5a9ffa15010924cad1e62ef206b08dfe3852aac3/db-tests/src/test/java/gov/nasa/jpl/aerie/database/PlanCollaborationTests.java#L190)
- [PlanCollaborationTests Reviewer](https://github.com/NASA-AMMOS/aerie/blob/5a9ffa15010924cad1e62ef206b08dfe3852aac3/db-tests/src/test/java/gov/nasa/jpl/aerie/database/PlanCollaborationTests.java#L202)
- TagsTest is implicitly used in the calls to `tagsHelper.insertTag()` in `PlanCollaboration.TagsTests`

Reply:
Ah, apologies - I was looking at the diff and didn't realize that these usernames predate this PR